doc/pylint.rst

```diff
@@ -1,35 +1,104 @@
+You can select the desired file to analyze directly in the :doc:`editor` by clicking anywhere within it.
+To run the analysis, press the configurable shortcut (:kbd:`F8` by default), select :menuselection:`Source --> Run code analysis` from the menu bar or click the :guilabel:`Analyze` button in the Code Analysis pane.
+If the Code Analysis pane is not visible, you can open it under :menuselection:`View --> Panes --> Code Analysis`.
 All standard checks are run by default.
-You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=<list of message names>`` comment at the respective level, where the ``<...>`` portion should be replaced with a comma-separated list (or single value) without the ``<``/``>`` of Pylint message names (*e.g.* ``multiple-statements``, or ``fixme, line-too-long``; you can also use error codes like ``C0321`` although names are preferred for clarity).
-Or, you can globally suppress certain messages by editing the :file:`.pylintrc` configuration file in your user home directory.
+To go directly to a line in the :doc:`editor` highlighted by a failed check, just click its name.
+
+.. image:: images/pylint/code-analysis-editor.gif
+   :alt: Spyder Pylint pane, showing running analysis and clicking failed check
+
+|
+
+You can also manually enter the path of a file you'd like it to check in the path entry box in the pane's toolbar.
+The analyzer works with both individual scripts and whole Python packages (directories containing an :file:`__init__.py` file).
+
+.. image:: images/pylint/code-analysis-file.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why. 
+If Pylint does succeed, the :guilabel:`Output` will show the raw plain text analysis results on the selected file, allowing you to easily browse and copy/paste the full message names and descriptions.
+
+.. image:: images/pylint/code-analysis-output.png
+   :align: center
+   :alt: Spyder Pylint pane, showing output
+
+|
+
+You can click the dropdown or press the dropdown arrow in the filename field to view results of previous analyses.
+
+.. image:: images/pylint/code-analysis-history.png
+   :align: center
+   :alt: Spyder Pylint pane, showing history
+
+============
+Options menu
+============
+The number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog, available from the :guilabel:`Code Analysis` options menu.
+
+.. image:: images/pylint/code-analysis-history-custom.gif
+   :alt: Spyder Pylint pane, customizing history dialog
+
+|
+
+You can also expand or collapse one or all the sections in the pane by using the corresponding options in the context menu.
```

Comment:
```suggestion
You can also expand or collapse one or all the sections in the pane by using the corresponding options in the options menu.
```

The gif is showing how to do that from the options menu, not the context one. Also, this section is about the options menu.

Reply:
Agreed.